From c29eb6ea86494dacaa98e6afd427d3899fb243ea Mon Sep 17 00:00:00 2001
From: David Rothenberger <daveroth@acm.org>
Date: Mon, 14 Sep 2015 14:39:28 +0300
Subject: [PATCH 16/22] Support swig3

Fugly hack to fix build with SWIG 3.0.x.  The pre-processor seems to
expand the %define correctly with only the change to %{..%}, but
%extend still breaks somehow.

https://github.com/swig/swig/issues/379
---
 subversion/bindings/swig/svn_delta.i | 2 --
 1 file changed, 2 deletions(-)

diff --git a/subversion/bindings/swig/svn_delta.i b/subversion/bindings/swig/svn_delta.i
index 36c776b..ae45e1f 100644
--- a/subversion/bindings/swig/svn_delta.i
+++ b/subversion/bindings/swig/svn_delta.i
@@ -207,8 +207,6 @@ void _ops_get(int *num_ops, const svn_txdelta_op_t **ops)
 
 #ifdef SWIGPYTHON
 %pythoncode %{
-# This function is for backwards compatibility only.
-# Use svn_txdelta_window_t.ops instead.
 svn_txdelta_window_t_ops_get = svn_txdelta_window_t._ops_get
 %}
 #endif
-- 
2.37.2

